src/detail-page/components/TabsContainer.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from 'react';
+import { Card, Tabs } from '@material-ui/core';
+import styled from 'styled-components';
+
+import { TabPanel } from './TabPanel';
+import { StyledTab } from './StyledTab';
+import { JobsTab } from './JobsTab';
+import { LearnTab } from './LearnTab';
+import { ResourcesTab } from './ResourcesTab';
+import { categories } from './TabCategories';
+
+const StyledTabs = styled(Tabs)`
```

Comment:
Also here we can rename import ( `import { Tabs as MuiTabs } from '@material-ui/core';`) and then our component can be named simply `Tabs`! :)

Reply:
Done!